vxdetector/Output_counter.py

```diff
@@ -18,58 +22,120 @@ def directory_navi(file_name, path, dir_name, dir_path):
+                                                               temp_path,
+                                                               all_reads, mode)
     if os.path.exists(new_file):
         header = True
     else:
         header = False
     with open(new_file, 'a', newline='') as o:
-        fieldnames = ['Read-file', 'Unaligned Reads [%]',
-                      'Sequenced variable region', 'Probability [%]']
+        fieldnames = ['Read-file', 'Number of Reads', 'Unaligned Reads [%]',
+                      'Not properly paired', 'Sequenced variable region', 'V1',
+                      'V2', 'V3', 'V4', 'V5', 'V6', 'V7', 'V8', 'V9',
+                      'Not aligned to a variable region']
         writer = csv.DictWriter(o, fieldnames=fieldnames)
         if header is False:
             writer.writeheader()
-        writer.writerow({
-            'Read-file': file_name,
-            'Unaligned Reads [%]': unaligned_count,
-            'Sequenced variable region': most_probable_V,
-            'Probability [%]': probability})
+        writer.writerow({'Read-file': file_name, 'Number of Reads': all_reads,
+                         'Unaligned Reads [%]': unaligned_count,
+                         'Not properly paired': unpaired,
+                         'Sequenced variable region': most_probable_V,
+                         'V1': dictionary['V1'], 'V2': dictionary['V2'],
+                         'V3': dictionary['V3'], 'V4': dictionary['V4'],
+                         'V5': dictionary['V5'], 'V6': dictionary['V6'],
+                         'V7': dictionary['V7'], 'V8': dictionary['V8'],
+                         'V9': dictionary['V9'],
+                         'Not aligned to a variable region': no_V})
+
+
+def print_output(dictionary, unaligned_count):
+    most_probable_V, dictionary = region_count(dictionary, unaligned_count,
+                                               temp_path=None, all_reads='',
+                                               mode='')
+    V1 = str(dictionary['V1'])
+    V2 = str(dictionary['V2'])
+    V3 = str(dictionary['V3'])
+    V4 = str(dictionary['V4'])
+    V5 = str(dictionary['V5'])
+    V6 = str(dictionary['V6'])
+    V7 = str(dictionary['V7'])
+    V8 = str(dictionary['V8'])
+    V9 = str(dictionary['V9'])
+    print(f'\n{str(unaligned_count)}% were unaligned.')
+    print(f'Of all the aligned Reads most were aligned to: {most_probable_V}')
+    print('The probabilities of all regions is as follows [%]:')
+    print(f'V1: {V1}\nV2: {V2}\nV3: {V3}\nV4: {V4}\nV5: {V5}\n \
+          V6: {V6}\nV7: {V7}\nV8: {V8}\nV9: {V9}\n')
 
 
-def count(count, temp_path, file_name, file_type, path, dir_name, dir_path):
-    dictionary = {}
-    BED_path = temp_path + 'BED.bed'
-    Log_path = temp_path + 'bowtie2.log'
+def count(temp_path, file_name, file_type, path, dir_name, dir_path, mode):
+    dictionary = {'V1': 0, 'V2': 0, 'V3': 0, 'V4': 0, 'V5': 0,
```

Comment:
we should have a global constant listing the variable region names. It would than be easy to loop through these regions. Should we add/remove one, we would only have to update in a single location!